src/EnergyPlus/UnitarySystem.cc

```diff
@@ -0,0 +1,12162 @@
+                this->designCoolingCapacity = initUnitarySystemsQActual;
+
+            } // from IF(UnitarySystem(UnitarySysNum)%CoolingCoilType_Num == Coil_CoolingWater .OR. Coil_CoolingWaterDetailed
+            if (this->heatingCoilType_Num == DataHVACGlobals::Coil_HeatingWater) {
+
+                //     set water-side mass flow rates
+                Real64 mdot = this->maxHeatCoilFluidFlow;
+                PlantUtilities::SetComponentFlowRate(mdot,
+                                                     this->heatCoilFluidInletNode,
+                                                     this->heatCoilFluidOutletNodeNum,
+                                                     this->heatCoilLoopNum,
+                                                     this->heatCoilLoopSide,
+                                                     this->heatCoilBranchNum,
+                                                     this->heatCoilCompNum);
+                //     simulate water coil to find operating capacity
+                WaterCoils::SimulateWaterCoilComponents(this->heatingCoilName, FirstHVACIteration, this->heatingCoilIndex, initUnitarySystemsQActual);
+                this->designHeatingCapacity = initUnitarySystemsQActual;
+
+            } // from IF(UnitarySystem(UnitarySysNum)%HeatingCoilType_Num == Coil_HeatingWater) THEN
+
+            if (this->heatingCoilType_Num == DataHVACGlobals::Coil_HeatingSteam) {
+
+                //     set water-side mass flow rates
+                Real64 mdot = this->maxHeatCoilFluidFlow;
+                PlantUtilities::SetComponentFlowRate(mdot,
+                                                     this->heatCoilFluidInletNode,
+                                                     this->heatCoilFluidOutletNodeNum,
+                                                     this->heatCoilLoopNum,
+                                                     this->heatCoilLoopSide,
+                                                     this->heatCoilBranchNum,
+                                                     this->heatCoilCompNum);
+                //     simulate steam coil to find operating capacity
+                SteamCoils::SimulateSteamCoilComponents(
+                    this->heatingCoilName,
+                    FirstHVACIteration,
+                    this->heatingCoilIndex,
+                    1.0,
+                    initUnitarySystemsQActual); // QCoilReq, simulate any load > 0 to get max capacity of steam coil
+                this->designHeatingCapacity = SteamCoils::GetCoilCapacity(
+                    DataHVACGlobals::cAllCoilTypes(this->heatingCoilType_Num), this->heatingCoilName, initUnitarySystemsErrorsFound);
+
+            } // from IF(UnitarySystem(UnitarySysNum)%HeatingCoilType_Num == Coil_HeatingSteam) THEN
+            if (this->suppHeatCoilType_Num == DataHVACGlobals::Coil_HeatingWater) {
+
+                //     set steam-side mass flow rates
+  
```

Comment:
@mbadams5 here's an example of where I need to access field names. Line 1401 of UnitarySystem.cc on this branch. This field is cooling supply air flow rate so I need std:string thisFieldName = N3fieldString.

    N3 , \field Cooling Supply Air Flow Rate
       \type real
       \units m3/s
       \minimum 0.0
       \autosizable
       \note Enter the magnitude of the supply air volume flow rate during cooling operation.
       \note Required field when Cooling Supply Air Flow Rate Method is SupplyAirFlowRate.
       \note This field may be blank if a cooling coil is not included in the unitary system.